### ISSUE TITLE
Module not found: Missing "./entities/workspace/security" specifier in "@scalar/oas-utils" package

### ISSUE THREAD

**Issue by hanspagel** [scalar/scalar#2226]
Fresh Vite + Vue project, installed and imported @scalar/api-reference, got:

```
✘ [ERROR] Missing "./entities/workspace/security" specifier in "@scalar/oas-utils" package [plugin vite:dep-pre-bundle]

    node_modules/.pnpm/@scalar+client-app@0.1.10_tailwindcss@3.4.4_typescript@5.5.2/node_modules/@scalar/client-app/dist/store/workspace.js:7:42:
      7 │ import { createSecurityScheme as z } from "@scalar/oas-utils/entities/workspace/security";
```

Try to fix it now.

**Comment by hanspagel** [comment]
I think this is fixed already, the next release should be good.

**Comment by GusevPM** [comment]
Hi! How can it be fixed?

**Comment by hanspagel** [comment]
Fixed in @scalar/api-reference 1.24.17! Just released this a minute ago. :v:
